src/components/Missions.js

```diff
@@ -1,8 +1,75 @@
 import React from 'react';
+import '../styles/Missions.css';
 
 function Missions() {
   return (
-    <div>Missions</div>
+    <main>
+      <table>
```

Comment:
The Div and main elements are both parent elements in this context. It would therefore be wise to contain them within a fragment.  You could kindly contain them between <> and </> 😃😃